R/if_switch_linter.R

```diff
@@ -14,13 +14,77 @@
 #'   approach is roughly linear in the number of conditions that need to
 #'   be evaluated, here up to 3 times).
 #'
+#' @param max_branch_lines,max_branch_expr Integer, default 0 indicates "no maximum".
```

Comment:
Plural the expr arg too? `max_branch_exprs`?

Reply:
that sounds unnatural to me, I would pluralize as "several expr nodes" or "several expressions", never "several exprs".

would you prefer max_branch_expressions?

Reply:
`max_branch_expressions` sounds good as well.
We might even drop the `_branch_` in both arg names for brevity, WDYT?

I read expr as a contraction of expression, so I don't have a problem with "exprs", but I can see the confusion with <expr> nodes.

Reply:
Thinking again, "expr" anchors too much on thinking in terms of XPath, which probably isn't the most user-friendly approach. "expressions" is more universal / understood without knowing any internals of the AST/its XML representation.

I'd keep `branch` for clarity.